packages/next-playwright/README.md

```diff
@@ -0,0 +1,37 @@
+# @next/playwright
+
+### Install `@next/playwright` in your project
+
+```sh
+npm install -D @next/playwright
+```
+
+### Update `playwright.config.ts`
+
+```javascript
+import { defineConfig } from '@playwright/test'
+
+export default defineConfig({
+  webServer: {
+    command: 'pnpm dev -- --experimental-test',
+    url: 'http://localhost:3000',
+  },
+})
+```
+
+### Use the `@next/playwright` to create tests
+
+```javascript
+import { expect } from '@playwright/test'
+import { test } from '@next/playwright'
+
+test('/product/shoe', async ({ page, next }) => {
+  next.mockFetchJson('http://my-db/product/shoe', {
```

Comment:
Could URL support paths without the full URL? When applicable.

Reply:
Hmm. Not sure I see a value immediately. Would that be for a local API route access?

Reply:
Ping on this to clarify.